CHANGELOG.md

```diff
@@ -15,25 +15,49 @@
 
 ## 4.0.0
 
-**v4.0 is a big release, so here are the main highlights:** (full list of changes below)
+### Highlights
```

Comment:
Looks good! Should we also highlight changes to the client (no JSON files returned by label, chatbot)?

I think we should also put a link to this in the guides at least temporarily. It will increase visibility.

Reply:
Thanks @freddyaboulton! I'll add the changes to the client.

Which guides are you thinking?